pkg/controller/fedvolumebackup/fed_volume_backup_control.go

```diff
@@ -130,6 +131,17 @@ func (c *defaultBackupControl) updateVolumeBackupMetrics(volumeBackup *v1alpha1.
 	}
 }
 
+// add zero to the backup status counters; this sets the backup counter metric to zero when the federated manager restarts.
+func (c *defaultBackupControl) addZeroToBackupStatusMetric(volumeBackup *v1alpha1.VolumeBackup) {
+	ns := volumeBackup.Namespace
+	tcName := volumeBackup.GetCombinedTCName()
+	metrics.FedVolumeBackupStatusCounterVec.WithLabelValues(ns, tcName, string(v1alpha1.VolumeBackupRunning)).Add(0)
```

Comment:
it's better to use `GetMetricWithLabelValues()` to create a new counter if it doesn't exist instead of adding a zero value to the counter.

Reply:
done